File: christmas/december task 2/function tasks/Factor.js
const prompt = require('prompt-sync')();

function factorOf(number){
    

  number = prompt("Enter any integer");
   
    factor = 0;

    for(count = 1; count <= number; count ++) {
        if(number % count == 0) {
          factor ++;

          
        }
       
        }
          

          return factor;  
    }


number = 0;
console.log(factorOf(number));

File: christmas/december task 2/function tasks/Factor.java
import java.util.Scanner;
    public class Factor {
        public static void main(String...Odili) {

            int number = 0;
        System.out.println(factorOf(number));
        }

    public static int factorOf(int number){
        Scanner input = new Scanner(System.in);

        System.out.println("Enter any integer");
            number = input.nextInt();

            int factor = 0;

        for(int count = 1; count <= number; count ++) {
            if(number % count == 0) {
              factor ++;

              
            }
           
        }
          

          return factor;  
    }


    }
